README.md

```diff
@@ -1,2 +1,5 @@
-# .github
-Default community health files
+# Default community health files
+
+These files are used by default for subprojects that don't provide their own.
+
+See [Github's community health files description](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file).
```

Comment:
GitHub's description of community health files

Reply:
done.